_posts/kevin_hartmann/2018-03-19-Drops-of-Jupyter.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "Drops of Jupyter..."
+breadcrumb: true
+author: kevin_hartmann 
+date: 2018-03-19
+categories: kevin_hartmann
+tags:
+    - Jupyterhub 
+    - Jupyter Notebooks 
+    - Docker 
+    - ansible 
+teaser:
+    info: "or, How I Learned To Stop Worrying and Dockerized Jupyterhub" 
+    image: kevin_hartmann/jupiter.jpg
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+<image src="{{ site.urlimg }}/kevin_hartmann/jupiter.jpg" alt="Weather vortex pattterns on planet Jupiter" />
+
+Hey, wait! That's the surface of **Jupiter**, the planet! We're talking about **Jupyter**, with a 'y', as in **Jupyter Notebooks** and their multi-user counterpart, **Jupyterhub**! 
+
+<image src="{{ site.urlimg }}/kevin_hartmann/jupyter.png" alt="Jupyter project logo" /> Yeah, those guys! That's much better!
+
+This blog post is about how I set up our Jupyterhub service at jupyterhub.cancercollaboratory.org. It's aimed at a technical audience, so if you don't have that kind of background, don't be afraid to skip over parts that you don't understand right away, and come back to them later. That's how I learned most of what I know about computers; as far as I know, it's how most people still do. With that said, let's carry on into our exploration into the wild and tumultuous world of Jupyterhub!
+
+Our Jupyterhub service is hosted on a virtual machine inside an Openstack cluster, and we naturally wanted to be able to re-build the whole thing from scratch without having to work too hard at it. We use *ansible* for automation of these sorts of things, so I created an ansible playbook to set up the virtual machine, and set up jupyterhub to run as a docker container within the virtual machine.
```

Comment:
Can you add a section on background regarding the hosting part, why one VM, and so on. Nothing large, just a short paragraph.